packages/lit-dev-content/samples/tutorials/carousel/00/before/motion-carousel.ts

```diff
@@ -0,0 +1,106 @@
+/* playground-fold */
+import {LitElement, html, PropertyValues, noChange} from 'lit';
+import {customElement, property, query} from 'lit/decorators.js';
+import {animate} from '@lit-labs/motion';
+import {styleMap} from 'lit/directives/style-map.js';
+import {styles} from './styles.js';
+
+@customElement('motion-carousel')
+export class MotionCarousel extends LitElement {
+  static styles = styles;
+
+  @query('slot[name="selected"]', true)
+  private selectedSlot!: HTMLSlotElement;
+
+  @query('slot[name="previous"]', true)
+  private previousSlot!: HTMLSlotElement;
+
+  private advancing = false;
+  private _selected = 0;
+  @property({type: Number})
+  get selected() {
```

Comment:
The transformer from TS -> JS doesn't handle decorated accessors, so this is transformed into the following JS:

```
__decorate(
  [property({type: Number})],
  MotionCarousel.prototype,
  'selected',
  null
);
```

Instead expected:

```
  static properties = {
    selected: {type: Number}
  }
```

Handling in the tutorial would require creating a `motion-carousel.js` file, or it could be addressed in the transformer.

Reply:
This is going to need a bug on lit's ts transformers

Reply:
lit/lit#2780

Reply:
Fixed.

Reply:
I see that a js file was already added for this but https://github.com/lit/lit.dev/pull/790 makes this work now so bring in the latest and reinstalling dependencies is an option too.